src/resource_clients/actor.js

```diff
@@ -66,36 +66,23 @@ class ActorClient extends ResourceClient {
     async call(options = {}) {
         ow(options, ow.object.exactShape({
             contentType: ow.optional.string,
-            memoryMbytes: ow.optional.number,
-            timeoutSecs: ow.optional.number,
+            memory: ow.optional.number,
+            timeout: ow.optional.number,
```

Comment:
Let's keep the `Mbytes` and `Secs`. It's in line with our naming. We always add the units to number variables.

Reply:
Ok, make sense. Renamed.